modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,156 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= Understanding {product-title} upgrade channels
-
-In {product-title} 4.1, Red Hat introduced the concept of upgrade channels for
-recommending the appropriate upgrade versions to your cluster. Upgrade channels
-separate upgrade strategies and also are used to control the cadence of updates.
-Channels are tied to a minor version of {product-title}. For instance,
-{product-title} {product-version} channels will never include an upgrade to a
-4.4 release. This ensures administrators make an explicit decision to upgrade to
-the next minor version of {product-title}. Channels only control updates and
-have no impact on the version of the cluster you install; the
+= {product-title} Upgrade Channels and Releases
+
+In {product-title} 4.1, Red Hat introduced the concept of Upgrade Channels for
+recommending the appropriate release versions for cluster upgrade. Upgrade Channels
+separate upgrade strategies and are also used to control the cadence of upgrades.
+Upgrade Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} Upgrade Channels will never include an upgrade to a
+4.5 release. This ensures administrators make an explicit decision to upgrade to
+the next minor version of {product-title}. Upgrade Channels only control release
+selection and have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+{product-title} {product-version} has the following Upgrade Channels to choose from:
+
+.  candidate-{product-version}
+.  fast-{product-version}
+.  stable-{product-version}
+
+= candidate-{product-version}
+
+The candidate-{product-version} channel will contain all z-stream ({product-version}.z)
+releases. If a given release is only available in the candidate-{product-version}
+channel and not available in either the fast-{product-version} or stable-{product-version}
+channels, it is considered a release candidate. Although release candidates contain
+all the features of the product, they are not supported and should only be used to
+test feature acceptance and assist in qualifying the next version of 
+{product-title} in your infrastructure. Additionally, a release candidate can only be
+upgraded to another release candidate version and, if one does not exist, is not
```

Comment:
You don't define "release candidate version".  I think the main issue this sentence is trying to convey is that we will happily leave dead-ends in the release candidate graph, while fast/stable have support-level guarantees about us (eventually) providing exits from any release in that channel to get you to the current channel tip.  That's a somewhat awkward point to address here, because we haven't talked about fast/stable yet.  I'd rather talk about stable, then fast, then candidate, because we expect most admins to only care about stable.  But if you want a narrower pivot, maybe reword this sentence to:

> Additionally, the candidate-{product-version} may include dead end releases from which there are never recommended upgrades.

or some such?